test-integration/helpers/ui/test-element.coffee

```diff
@@ -94,10 +97,12 @@ class TestHelper extends TestItemHelper
       defaultWaitTime: 40 * 1000 # TODO: Letting tests define their own wait time is dangerous. tutor-dev takes > 10sec to delete a task-plan
       # defaultWaitTime: 20 * 1000 # 20sec seems to be enough for deployed code but not local
 
+    @test = test
     @_options = _.assign {}, defaultOptions, options
     @_el = {}
 
     commonElements.loadingState = @options.loadingLocator
+    commonElements.self = testElementLocator
```

Comment:
Hmm...but then would `loadingState` also be passed in through `COMMON_ELEMENTS`?  It seems to me that since these -- `self` and `loadingState` -- are required, they should be different from the others, as opposed to saying that `COMMON_ELEMENTS` must have certain locators.  This way, there's no demands on `COMMON_ELEMENTS` to have any specifically named locators, and the potential for them to be misnamed.